prov/efa/src/efa_fabric.c

```diff
@@ -467,10 +467,22 @@ static int efa_alloc_fid_nic(struct fi_info *fi, struct efa_context *ctx,
 }
 
 #if HAVE_LIBCUDA
+/*
+ * efa_get_gdr_support() check if GPUDirect RDMA is supported by
+ * reading from sysfs file "class/infiniband/<device_name>/gdr"
+ * and set content of gdr_support accordingly.
+ *
+ * Return value:
+ *   return 1 if sysfs file exist and has 1 in it.
+ *   return 0 if sysfs file does not exist or has 0 in it.
+ *   return a negatie value if error happened.
+ */
 static int efa_get_gdr_support(char *device_name)
 {
+	static const int MAX_GDR_SUPPORT_STRLEN = 80;
```

Comment:
where did this come from?  Not sure I love dropping an 80 byte variable on the stack.

Reply:
Not sure what is the good value, I changed to 8 in new revision.